test/pull/expected-stdout

```diff
@@ -12,7 +12,7 @@ INFO	  +  R bar/extractor/keboola.ex-db-mysql/with-rows/rows/disabled
 INFO	  +  R bar/extractor/keboola.ex-db-mysql/with-rows/rows/test-view
 INFO	  +  R bar/extractor/keboola.ex-db-mysql/with-rows/rows/users
 INFO	  +  C bar/other/keboola.variables/python-transformation-shared-code-variables
-INFO	  +  R bar/other/keboola.variables/python-transformation-shared-code-variables/rows/default-values
+INFO	  +  R bar/other/keboola.variables/python-transformation-shared-code-variables/values/default-values
```

Comment:
Tieto zmeny uz ovplyvnuju tento funkcionalny test, ... a to cestu ku `VariablesValuesRow`.
Cesta k samotnemu `variables` configu este nie je upravena, ... kedze pri nacitavani objektov z API sa este medzi nimi nevytvaraju `Relations`.